layers/meta-resin-chip/recipes-support/resin-udevmount/resin-udevmount.bb

```diff
@@ -0,0 +1,16 @@
+SUMMARY = "udev rules for UBIFS"
+LICENSE = "CLOSED"
+PR = "r1"
```

Comment:
We add a new recipe so PR r0 should be the first revision which is the default value so we can remove it altogether.